README.md

```diff
@@ -65,7 +65,8 @@ const createAccountLimiter = RateLimit.middleware({
   timeWait: 3*1000, // slow down subsequent responses by 3 seconds per request
   max: 5, // start blocking after 5 requests
   prefixKey: 'post/user', // to allow the bdd to Differentiate the endpoint 
-  message: "Too many accounts created from this IP, please try again after an hour"
+  message: "Too many accounts created from this IP, please try again after an hour",
+  messageKey: "error"
```

Comment:
maybe set it as message since it is the default behaviour

Reply:
Yeah makes sense, updated!

Reply:
cool!

Is this urgent for you? If it is, I can release it asap otherwise I'll just wait the resolution of #47 and make a release with both changes

Reply:
It's ok, np